### PR TITLE
ENH: Use suggested landmark names

### PR DESCRIPTION
**Pull request by allemangD**
When placing landmarks in order, use the landmark labels from the table. This is assuming landmarks are placed in the same order as in the table.

Resolves https://github.com/DCBIA-OrthoLab/Q3DCExtension/issues/53

### CODE REVIEW

**Comment by bpaniagua** [comment]
@jcfr I am afraid I do not really understand the code that @allemangD committed. It looks good to me, but I would feel more comfortable if you have a second look. Thanks.

**Comment by allemangD** [comment]
@bpaniagua in that case I will also add some comment to the code to make it easier to understand.

**Review by bpaniagua — APPROVED** [review]
Thank you, comments are really helpful. Looks good to me. Good to merge.

File: Q3DC/Q3DC.py [hunk 1]
```diff
@@ -1101,6 +1105,17 @@ def connectLandmarks(self, modelSelector, landmarkSelector, onSurface):
...
+
+        try:
+            # Find the index of the last-placed landmark and get the landmark label at that position.
+            # Ex. if the last-placed landmark was at the 3rd position, we want to use the 3rd landmark label.
+            n = obj.GetNumberOfMarkups()
+            label, description = self.current_suggested_landmarks[n - 1]
+            obj.SetNthMarkupLabel(n - 1, label)
+        except IndexError:
+            # If there are more landmarks than suggested labels then fetching the label would fail.
+            print('Not changing label; wrong number of markups.')
```
Comment (jcfr) [inline Q3DC/Q3DC.py:1117]: Considering using `logging.error()`.  See https://www.slicer.org/wiki/Documentation/Nightly/Developers/Style_Guide#Logging

Comment (jcfr) [inline Q3DC/Q3DC.py:1117]:
```suggestion
            logging.error('Not changing label; wrong number of markups.')
```

**Comment by allemangD** [comment]
I've replaced all the print-debugging statements with appropriate log messages. I think this is ready to merge.

**Review by jcfr — APPROVED** [review]

**Pull request merged**